common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/IdState.scala

```diff
@@ -5,7 +5,7 @@ sealed trait IdState {
   def maybeCanonicalId: Option[String]
   def allSourceIdentifiers: List[SourceIdentifier]
 }
-
+sealed trait pff extends IdState
```

Comment:
Of course this isn't going to be called `pff` but I'm struggling to come up with a name. Suggestions would be appreciated